requirements.txt

```diff
@@ -3,8 +3,31 @@ matplotlib
 numpy
 tensorflow
 opencv-python>=4.5.1
+git+https://github.com/philferriere/cocoapi.git#egg=pycocotools&subdirectory=PythonAPI
```

Comment:
why this? brief google suggests that pycocotools might be hard to install on windows. for me I've always just run  pip install pycocotools and the specific HPE algorithms are always what pull it in.

it's also slightly confounded in my memory since one tracking algorithm did require a cocotools fork, but I don't think it is one that is frequently used (and creates a really annoying dependency conflict).

Reply:
I'll remove from here. This add might have been specific to OSX. Next time I hit this conflict issue, I'll add notes to the readme about resolving rather than adding as an additional requirement.